packages/onchain-role-enrolment/test/truffle/claim-manager-upgrade-test.ts

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
I think `test/upgrades` is more appropriate name then `test/truffle`

Reply:
`test/truffle` gives more intuition that is should be tested by `truffle`, IMO.